src/test/resources/application.conf

```diff
@@ -0,0 +1,28 @@
+# Copyright 2023 HM Revenue & Customs
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+include "frontend.conf"
+appName = "test-application"
+
+play.modules.enabled += "uk.gov.hmrc.play.bootstrap.FrontendModule"
+play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientModule"
+play.modules.enabled += "uk.gov.hmrc.play.audit.AuditModule"
+play.modules.enabled += "uk.gov.hmrc.play.bootstrap.graphite.GraphiteMetricsModule"
```

Comment:
is this file required ?

Reply:
Yes otherwise there are 3 Errors and 1 failure printed to console

`[info] *** 3 SUITES ABORTED ***`
`[info] *** 1 TEST FAILED ***`

`[info] WrapperServiceSpec:
[info] service.WrapperServiceSpec *** ABORTED ***
[info]   com.google.inject.ProvisionException: Unable to provision, see the following errors:
[info] 
[info] 1) No implementation for com.kenshoo.play.metrics.MetricsFilter was bound.
[info]   while locating com.kenshoo.play.metrics.MetricsFilter
` etc....